doc/man1/openssl.pod

```diff
@@ -691,113 +691,20 @@ See L<property(7)> for a more detailed description.
 
 =head1 ENVIRONMENT
 
-The OpenSSL library can be take some configuration parameters from the
-environment.  Some of these variables are listed below.  For information
-about specific commands, see L<openssl-engine(1)>,
-L<openssl-rehash(1)>, and L<tsget(1)>.
-
-For information about the use of environment variables in configuration,
-see L<config(5)/ENVIRONMENT>.
-
-For information about querying or specifying CPU architecture flags, see
-L<OPENSSL_ia32cap(3)>, L<OPENSSL_s390xcap(3)> and L<OPENSSL_riscvcap(3)>.
+The OpenSSL libraries can be take some configuration parameters from the
```

Comment:
Since we're changing this paragraph, can we fix the grammar? It should probably read
```suggestion
The OpenSSL libraries can take some configuration parameters from the
```

but I'm not sure about the use of `some`. If there's only a proper subset of the config params that can be taken from the environment, then `some` is correct, otherwise it should be removed.

Reply:
Done.
In my view the `some` is correct because there are many config params that can not be taken from the environment, namely all static configuration.

Reply:
Thanks